src/Funding/SFA.DAS.Funding.Api.UnitTests/SFA.DAS.Funding.Api.UnitTests.csproj

```diff
@@ -24,7 +24,6 @@
     </ItemGroup>
 
     <ItemGroup>
-      <Folder Include="Controllers\" />
```

Comment:
seems like a test file has been added to this folder? Did you mean to delete Models?